info/available-rules.md

```diff
@@ -1,131 +1,131 @@
+| 1    | 1.1.1    | EXCEPTION_SUFFIX                          | Check: warns if a class that extends any Exception class does not have an Exception suffix<br>Fix: Adding suffix "Exception" to a class name                                                                                                                      | yes | no                                                                                                                                                                                                                         | Need to add tests for this                                                                                                                                                                                                    |
+| 1    | 1.1.1    | IDENTIFIER_LENGTH                         | Check: identifier length should be in range [2,64] except names that are used in industry like {i, j} and 'e' for catching exceptions.<br>Fix: no fix is available to automatically create a name                                                         | no  | no                                                                                                                                                                                                                         | May be make this rule configurable (length)                                                                                                                                                                                   |
+| 1    | 1.1.1    | FILE_NAME_INCORRECT                       | Check: warns if a file name does not have an extension .kt/.kts<br>Fix: no                                                                                                                                                                                        | no  | no                                                                                                                                                                                                                         | Extensions should be configurable; it can be autofixed                                                                                                                                                             |
```

Comment:
Check: warns if a file name does not have a `.kt`/`.kts` extension.

Fix: **none**.

Reply:
OK